unittests/CMakeLists.txt

```diff
@@ -126,19 +126,34 @@ target_link_libraries(test_spin
   gtest_main)
 gtest_discover_tests(test_spin)
 
-# build the test qudit execution manager
-add_subdirectory(qudit)
-add_executable(test_qudit main.cpp qudit/SimpleQuditTester.cpp)
+# # build the test qudit execution manager
```

Comment:
We want to keep the SimpleQuditTester here.

Reply:
Uncommented